.github/workflows/mutation-tests-core.yml

```diff
@@ -0,0 +1,63 @@
+name: mutation-tests-core
```

Comment:
Rather than two workflows with one job each, we should have one workflow with two jobs. Then there's less duplicative code in the workflow files.

In fact, using a matrix (like the build does for the operating system) would only need one job to be defined, which would then fan out for the two sets of mutation tests.

Reply:
Yes, I definitely agree that putting them in one workflow can reduce some duplication in workflow.

I put them in this way was because in @martintmk issue description mentioned there are two workflows.

I thought initially maybe we want to put just core as required and legacy as optional in the future when we don't support legacy.

If we can confirm that we need just one workflow I can make the change.

Reply:
Just one workflow with two jobs which specify which set to run via a matrix please.

Reply:
Updated.